pkg/controllers/csi/metadata/models.go

```diff
@@ -14,20 +14,20 @@ type TimeStampedModel struct {
 
 // TenantConfig holds info about a given configuration for a tenant.
 type TenantConfig struct {
-	CodeModule CodeModule `gorm:"foreignKey:Version;references:DowloadedCodeModuleVersion"`
+	CodeModule CodeModule `gorm:"foreignKey:DowloadedCodeModuleVersion;references:Version"`
```

Comment:
do we need that? seems like "doppeltgemoppelt" to me as Version is already declared primary key in the CodeModule struct?

Reply:
we need, because otherwise it won't find column, column name doesn't fit into GORM convention name for FK.

Reply:
double checked we need it,otherwise GORM failing:
```
2024/03/05 10:50:08 /Users/andrii.soldatenko/work/dynatrace-operator/pkg/controllers/csi/metadata/sqlite_test.go:674 FOREIGN KEY constraint failed
[0.772ms] [rows:0] INSERT INTO `tenant_configs` (`created_at`,`updated_at`,`deleted_at`,`uid`,`name`,`downloaded_code_module_version`,`config_dir_path`,`tenant_uuid`,`max_failed_mount_attempts`) VALUES ("2024-03-05 10:50:08.341","2024-03-05 10:50:08.341",NULL,"","dynakube","1.288.0.20240227-104015","/data/mts86258/dynakube/config","mts86258",10)
FOREIGN KEY constraint failed
<nil>
```